### ISSUE TITLE
More than two linked charts?

### ISSUE THREAD

**Issue by hrbrmstr** [mozilla/metrics-graphics#309]
More confirming current functionality than requesting a feature or issuing a bug report. Is the intent for linked charts just to link 2 charts, or should it work with multiple ones?

**Comment by almossawi** [comment]
It is to link two charts, as in the [examples](http://metricsgraphicsjs.org/examples.htm). There's an open issue for the other feature: https://github.com/mozilla/metrics-graphics/issues/241.

**Comment by hamilton** [comment]
@hrbrmstr Ali and I talked when we were making this feature about having the option of putting a set of charts in some sort of group, so that all the charts in that group trigger the link function. We needed the current functionality for an internal project so we developed it to meet that specification.

Would be happy to revisit this and make more fleshed-out linking functionality for a near-future version. I can see clear use cases.
